src/Member.ts

```diff
@@ -7,36 +7,70 @@ import { Permissions } from './Permissions'
 import { Sync } from './Sync'
 import { randomBytes } from 'crypto'
 import { Timestamp } from '@consento/hlc'
-import { Feed } from './Feed'
+import { Feed, FeedLoader, defaultFeedLoader } from './Feed'
 
 export interface MemberOptions {
   id?: ID
   initiator?: ID
+  loadFeed?: FeedLoader
 }
 
 export class Member {
-  readonly feed: Feed
-  readonly syncState: Sync
-  readonly id: ID
-  readonly initiator: ID
+  _feed?: Feed
+  _syncState?: Sync
+  readonly _id: ID
+  initiator: ID
+  private readonly loadFeed: FeedLoader
+
+  static async create (opts?: MemberOptions): Promise<Member> {
+    const member = new Member(opts)
+    await member.init()
+
+    return member
+  }
 
   constructor ({
     id = randomBytes(8).toString('hex'),
-    initiator = id
+    initiator = id,
+    loadFeed = defaultFeedLoader
```

Comment:
Looking over the code the name `loadFeed` has been sticking out to me: As a developer using this API: what am I supposed to do? In the current case its like `getOrLoadFeed` if I understand it correctly but I was wondering if maybe `reqestFeed` may be appropriate?